lib/etc/openturns.conf.in

```diff
@@ -975,9 +975,7 @@
   <Distribution-RoughnessSamplingMethod           value_int="MonteCarlo"/>
   <Distribution-RoughnessSamplingSize             value_int="524288"  />
   <Distribution-SmallDimensionRoughness           value_int="3"       />
-
-  <!-- OT::DiscreteDistribution parameters -->
-  <DiscreteDistribution-SupportEpsilon value_float="1.0e-14" />
+  <Distribution-SupportEpsilon            value_float="1.0e-14" />
```

Comment:
I dont think it will actually solve #1242 as the circular dependency is still there isnt it ?

Reply:
Indeed, I removed this.